InnerEye/ML/run_ml.py

```diff
@@ -483,26 +498,26 @@ def run_inference_for_lightning_models(self, checkpoint_paths: List[Path]) -> No
             if torch.distributed.is_initialized():
                 torch.distributed.destroy_process_group()
             trainer, _ = create_lightning_trainer(self.container, num_nodes=1)
+            self.container._model = type(self.container.model).load_from_checkpoint(checkpoint_path=str(checkpoint_paths[0]))
```

Comment:
Can we maybe make that a method on the container, so that we don't need to mess with the container's private fields?